### ISSUE TITLE
Dependency test-utils should have scope test

### ISSUE THREAD

**Issue by rvaldeolmillos** [mikolajmitura/java-properties-to-json#63]
Hi,
First thank you for sharing your work, it has been very useful for me.

I think the following dependency should have an scope "test", don't you agree? This would reduce the footprint of final project eliminating unneeded transitive dependencies.
```
        <!-- tests -->
        <dependency>
            <groupId>pl.jalokim.utils</groupId>
            <artifactId>test-utils</artifactId>
            <version>${java-utils.version}</version>
        </dependency>
```

**Comment by mikolajmitura** [comment]
Hi, 
Thanks for issue and for usage. You right this dependency should be in test scope.

**Comment by mikolajmitura** [comment]
I have released 5.2.2 version.
